### PR TITLE
remove some error suppressions

### PR DESCRIPTION
**Pull request by handeyeco**
## Summary:
idk, just following some threads by un-excepting errors.

### CODE REVIEW

File: config/test/test-setup.ts [hunk 1]
```diff
@@ -35,7 +35,6 @@ if (typeof window !== "undefined") {
     // Override the window.location implementation to mock out assign()
     // We need to access window.location.assign to verify that we're
     // redirecting to the right place.
-    /* eslint-disable no-restricted-syntax */
```
Comment (handeyeco) [inline config/test/test-setup.ts:38]: Some file-wide exceptions seemed to be for nothing, but our linter doesn't seem to find useless file-wide exceptions.

File: packages/perseus-editor/src/multirenderer-editor.tsx [hunk 2]
```diff
@@ -31,8 +31,7 @@ import type {
 
     // Multi-item item types
     Item,
-    // ItemTree is used below, the linter is confused.
```
Comment (handeyeco) [inline packages/perseus-editor/src/multirenderer-editor.tsx:34]: Doesn't seem confused anymore

File: packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx [hunk 3]
```diff
@@ -5,9 +5,6 @@ import * as React from "react";
 
 import {flags} from "../../../__stories__/flags-for-api-options";
 
-// Disabling the following linting error because the import is needed for mocking purposes.
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-import examples from "./locked-function-examples";
```
Comment (handeyeco) [inline packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx:10]: idk, tests seemed to be fine without it?

File: packages/perseus/src/widgets/orderer/orderer.tsx [hunk 4]
```diff
@@ -300,11 +295,6 @@ class Card extends React.Component<CardProps, CardState> {
     }
 }
 
-const NORMAL = "normal";
-const AUTO = "auto";
-const HORIZONTAL = "horizontal";
-const VERTICAL = "vertical";
```
Comment (handeyeco) [inline packages/perseus/src/widgets/orderer/orderer.tsx:306]: Two of these weren't used at all. Looking into it more it seemed like we could just use a string union.

**Review by jeremywiebe — APPROVED** [review]

**Pull request merged**